tests/resources/test_tasks_resources.py

```diff
@@ -74,10 +75,21 @@ def test_task_creation(
         "writers": [{"type": "test"}]
     }
 
-    resp = client.post(
-        "/vocabularies/tasks", headers=h, data=json.dumps(task_config)
-    )
-    assert resp.status_code == 202
-    p_read.assert_called()
-    p_apply.assert_called()
-    p_write.assert_called()
+    with patch.object(
```

Comment:
not a fan of chaining ctx managers... but I didn't find any other way... open to suggestions 😇

Reply:
I think you can include multiple comma-separated ctx managers as well in one `with ...` like:

```python
with patch.object(...) as p_read, \
        patch.object(...) as p_apply, \
        patch.object(...) as p_write:
    resp = ...
```

In [Python 3.10 you can even have parentheses](https://docs.python.org/3.10/whatsnew/3.10.html#parenthesized-context-managers) to avoid the ugly backslashes :)

Reply:
Thanks! done, without parenthesis to give support to older than 3.9